src/gsi/write_all.F90

```diff
@@ -121,6 +122,9 @@ subroutine write_all(increment)
   if (regional) then
      if (fv3_regional) then
         call wrfv3_netcdf(bg_fv3regfilenameg)
+        !if (fv3_cmaq_regional) then
```

Comment:
Same question as in the read, if this isn't needed, remove, don't comment out.

Reply:
will remove.